org.lflang/src/org/lflang/FileConfig.java

```diff
@@ -434,34 +435,132 @@ public static void copyDirectory(Path src, Path dest) throws IOException {
     }
 
     /**
-     * Copy a given file from 'src' to 'dest'.
+     * Copy a given file from 'source' to 'destination'.
      *
      * @param source The source file path string.
-     * @param dest The destination file path string.
+     * @param destination The destination file path string.
      * @throws IOException if copy fails.
      */
-    public static void copyFile(String src, String dest)  throws IOException {
+    public static void copyFile(String source, String destination)  throws IOException {
         try {
-            copyFile(Paths.get(src), Paths.get(dest));
+            copyFile(Paths.get(source), Paths.get(destination));
         } catch (IOException e) {
             throw e;
         }
     }
 
     /**
-     * Copy a given file from 'src' to 'dest'.
+     * Copy a given file from 'source' to 'destination'.
      *
      * @param source The source file path.
-     * @param dest The destination file path.
+     * @param destination The destination file path.
      * @throws IOException if copy fails.
      */
-    public static void copyFile(Path src, Path dest)  throws IOException {
+    public static void copyFile(Path source, Path destination)  throws IOException {
         try {
-            Files.copy(src, dest, java.nio.file.StandardCopyOption.REPLACE_EXISTING);
+            Files.copy(source, destination, java.nio.file.StandardCopyOption.REPLACE_EXISTING);
         } catch (IOException e) {
             throw e;
         }
     }
+    
+    /**
+     *  Lookup a file in the classpath and copy its contents to a destination path 
+     *  in the filesystem.
+     * 
+     *  This also creates new directories for any directories on the destination
+     *  path that do not yet exist.
+     * 
+     *  @param source The source file as a path relative to the classpath.
+     *  @param destination The file system path that the source file is copied to.
+     * @throws IOException 
+     */
+    public void copyFileFromClassPath(String source, String destination) throws IOException {
```

Comment:
Should we make the source a `URI` (or `URL`?) and the destination a `Path`?

Reply:
And shouldn't this be a static method?

Reply:
I don't think it can be static since it relies on `this.getClass()` to actually look for the file.

Reply:
> Should we make the source a `URI` (or `URL`?) and the destination a `Path`?

I think that is potentially a really nice feature.

Reply:
I think this is better left as to do since if we make this change, we need to add tests for it as well.
This change can go in conjunction with changing the files search algorithm as part of a `files` improvement effort.